tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -140,6 +181,38 @@ def install_rancher_self_signed(repo=RANCHER_HELM_REPO):
     time.sleep(120)
 
 
+def create_tls_secrets(valid_cert):
+    cert_path = DATA_SUBDIR + "/tls.crt"
+    key_path = DATA_SUBDIR + "/tls.key"
+    ca_path = DATA_SUBDIR + "/cacerts.pem"
+
+    if valid_cert == True:
+        # write files from env var
+        write_encoded_certs(cert_path, RANCHER_VALID_TLS_CERT)
+        write_encoded_certs(key_path, RANCHER_VALID_TLS_KEY)
+    else:
+        write_encoded_certs(cert_path, RANCHER_BYO_TLS_CERT)
+        write_encoded_certs(key_path, RANCHER_BYO_TLS_KEY)
+        write_encoded_certs(ca_path, RANCHER_PRIVATE_CA_CERT)
+
+    tls_command = export_cmd + " && kubectl -n cattle-system create secret tls tls-rancher-ingress --cert=" + cert_path + " --key=" + key_path
+    ca_command = export_cmd + " && kubectl -n cattle-system create secret generic tls-ca --from-file=" + ca_path
+
+    print(tls_command)
+    out = run_command_with_stderr(tls_command)
+    print(out)
+
+    if valid_cert == False:
+        print(ca_command)
+        out = run_command_with_stderr(ca_command)
+        print(out)
```

Comment:
I think we can remove all 4 of these print statements since `run_command_with_stderr` already outputs those by default

Reply:
Removed all of the same pattern from the entire file. Didn't realize we had that in the method already.